src/views/product-downloads-view/index.tsx

```diff
@@ -93,11 +93,20 @@ const ProductDownloadsViewContent = ({
 
 	// Group the selected release downloads by OS, for use in multiple places
 	const selectedRelease = releases.versions[currentVersion]
-	const downloadsByOS = useMemo(
+	let downloadsByOS = useMemo(
 		() => sortPlatforms(selectedRelease),
 		[selectedRelease]
 	)
 
+	if (currentProduct.slug === 'vagrant') {
+		downloadsByOS = {
+			...downloadsByOS,
+			linux: {
+				amd64: downloadsByOS?.linux?.amd64 || null,
+			},
+		}
```

Comment:
This will filter and prevent an error from being thrown when the user changes to a version that does not have a linux build.

```suggestion
		downloadsByOS = {
			...downloadsByOS,
			...(downloadsByOS.linux?.amd64
				? { linux: { amd64: downloadsByOS.linux.amd64 } }
				: {}),
		}
```